documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md

```diff
@@ -0,0 +1,258 @@
+
+In particular, it would be helpful to review our approach to the following:
+
+- **Systems**
+  - Heartbeat monitoring of internal services and public urls to ensure that the website is live and publicly accessible
+  - Automated notifications when:
+    - services are down
+    - 4xx or 5xx requests reach a certain threshold
+    - resources reach a certain usage threshold or are auto-scaled
+  - Logging capabilities (e.g. search, filtering) and retention policies to assist with troubleshooting
+- **Procedures**
+  - Internally-defined service level agreements for availability and incident response
+  - Documented incident response plan with steps and roles identified
+  - Documented plan for receiving and responding to alerts
+  - Staff trainings, tabletop and/or dry-runs for common types of incidents
+
+### Communication strategy
+
+The two main communication outputs needed for this deliverable are:
+
+- An email campaign that is sent to the subset of the grants.gov mailing list with whom we plan to share the site.
+- A communications plan that outlines future plans for stakeholder engagement about the Simpler.Grants.gov initiative. This plan should answer the following questions:
+  - Which stakeholder groups are we planning to engage?
+  - Who is responsible for managing the relationship with each group?
+  - What is the primary channel and cadence for those communications?
+
+Both the initial email communication and the communication plan should be reviewed and approved by key stakeholders in the recurring communications meetings.
+
+### Email subscription mechanism
+
+One stretch goal for this deliverable is to adopt a service that allows site visitors to subscribe to a mailing list to receive updates about the Simpler.Grants.gov initiative. As part of this deliverable the team should answer the following questions:
+
+- Which email marketing provider will we use for email-based communications?
+- How will visitors submit their information to be added to a mailing list for Simpler.Grants.gov communications?
+
+### Definition of done
+<!-- Required -->
+
+#### Must have
+
+- [ ] Basic requirements
+  - [ ] Code is deployed to main & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] All new services have passed a security review (as needed)
+  - [ ] All new services have completed a 508 compliance review (as needed)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly (as needed)
+- [ ] Functional requirements
+  - [ ] The static site has been updated with content approved by internal stakeholders
+  - [ ] The front-end infrastructure can handle site traffic of up to 1,000 concurrent users
+  - [ ] Tickets and/or documentation have been created that describe the infrastructure changes needed to support future site traffic at the following levels:
```

Comment:
Hmm, I'm wondering if this is essential or more of a stretch goal?

Reply:
Agreed, it's not mission critical for the launch. I moved this down into the "nice-to-have" section in [this commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/bcf247f3b4986382ba7e7c64e9d411d83842b0ae)